### PR TITLE
use Laravel's filesystem abstraction directly

### PR DESCRIPTION
**Pull request by emmachughes**
This removes the use of an unnecessary Docker volume (`/buckets/main_bucket`), when the default storage path in Laravel would have been fine. The change allows us to easily browse uploaded files in our dev environments.

`ContentAuthorStorage` will be replaced in favour of using Laravel's filesystem stuff directly.

Also, this fixes unit tests destroying the so-called bucket, which would remove all the files in your dev environment.

### CODE REVIEW

File: sourcecode/apis/contentauthor/tests/Integration/Article/ArticleFileVersionerTest.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 use App\Http\Libraries\ArticleFileVersioner;
 use App\Libraries\DataObjects\ContentStorageSettings;
 use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Storage;
 use Illuminate\Support\Str;
 use Tests\TestCase;
 use Tests\Helpers\ContentAuthorStorageTrait;
```
Comment (chrieinv) [inline sourcecode/apis/contentauthor/tests/Integration/Article/ArticleFileVersionerTest.php:13]: Is the trait required?

File: sourcecode/apis/contentauthor/tests/Integration/Jobs/PingVideoApiTest.php [hunk 2]
```diff
@@ -37,7 +37,7 @@ public function setUp(): void
         parent::setUp();
         $this->setUpContentAuthorStorage();
```
Comment (chrieinv) [inline sourcecode/apis/contentauthor/tests/Integration/Jobs/PingVideoApiTest.php:38]: Is the `ContentAuthorStorageTrait` required?

File: sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/API/H5PImportControllerTest.php [hunk 3]
```diff
@@ -78,7 +78,7 @@ public function importH5P()
             $this->_setUp();
             $this->setupAdapter(false, false);
 
-            $fakeDisk = Storage::fake($this->contentAuthorStorage->getBucketDiskName());
```
Comment (chrieinv) [inline sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/API/H5PImportControllerTest.php:81]: Is the `ContentAuthorStorageTrait` required?

File: sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/H5PExportTest.php [hunk 4]
```diff
@@ -33,7 +33,7 @@ protected function setUp(): void
         $this->setUpContentAuthorStorage();
```
Comment (chrieinv) [inline sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/H5PExportTest.php:33]: Is the `ContentAuthorStorageTrait` required?

File: sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/h5pTest.php [hunk 5]
```diff
@@ -8,6 +8,7 @@
 use H5PCore;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
 use RecursiveDirectoryIterator;
 use RecursiveIteratorIterator;
 use Tests\Helpers\ContentAuthorStorageTrait;
```
Comment (chrieinv) [inline sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/h5pTest.php:14]: Is the `ContentAuthorStorageTrait` required?

**Review by chrieinv — CHANGES_REQUESTED** [review]

**Comment by emmachughes** [comment]
ContentAuthorStorageTrait is gone now.

**Review by chrieinv — APPROVED** [review]

**Pull request merged**